eventrouter.go

```diff
@@ -52,6 +53,12 @@ type EventRouter struct {
 	// event sink
 	// TODO: Determine if we want to support multiple sinks.
 	eSink sinks.EventSinkInterface
+
+	// Keeps track of the last time the SharedInformer executed a re-sync
+	lastReset time.Time
+
+	// Indicates if this is the first time we are reset.
+	firstReset bool
```

Comment:
If I am reading this correctly don't firstReset and lastReset.IsZero() essentially represent the same thing? LastReset being the zero epoch means it has never been reset.

Reply:
You are correct.  I was testing some potential unpredictability.  `firstReset` should no longer be necessary.